scripts/prepare_lambda_deployment.py

```diff
@@ -70,6 +77,56 @@ def patch_cloudwatch_resource(resource_type_name, property_name):
     return input_json
 
 
+def patch_resource_names(input_json: JSON) -> JSON:
+    # Some Chalice-generated resources have named ending in `-event`. The
```

Comment:
This should be a docstring.

Might help to add a doctest to illustrate this more.